pkg/util/prioritymap/map_test.go

```diff
@@ -158,3 +158,132 @@ func TestArgumentSlice(t *testing.T) {
 
 	assert.Equal(t, expectedArgs, argMap.AsKeyValueStrings())
 }
+
+func TestDuplicateArguments(t *testing.T) {
+	defaultArgs := []string{
+		"--set-proxy=127.0.0.1",
+		"--set-host-id-source=auto",
+		"--set-server=localhost",
+		"--set-host-property=prop1",
+		"--set-host-property=prop2",
+		"--set-host-property=prop3",
+		"--set-host-property=prop3",
+		"--set-host-property=prop3",
+	}
+
+	custArgs := []string{
```

Comment:
nitpick:
my 🐒 🧠  needed like 0.5 sec to understand that `cust` == `custom`, I think just calling them `customArgs` would be better

also you did not shorten `defaultArgs` but `len("default") > len("custom")` 😛

Reply:
Fixed in [59eac56](https://github.com/Dynatrace/dynatrace-operator/pull/4357/commits/59eac56f63442a4fcf2850af4cf456725dd37ac0)